src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1017,8 +1016,9 @@ private function registerWorkflowConfiguration(array $config, ContainerBuilder $
             }
         }
 
-        $commandDumpDefinition = $container->getDefinition('console.command.workflow_dump');
-        $commandDumpDefinition->setArgument(0, $workflows);
+        $workflowLocator = (string) ServiceLocatorTagPass::register($container, $workflowLocator);
+        $container->setAlias('workflows', $workflowLocator);
+        $container->registerAliasForArgument($workflowLocator, ServiceLocator::class, 'workflows');
```

Comment:
what about `Psr\Container\ContainerInterface instead`? is there any reason to force `ServiceLocator`?

but more importantly: what about removing this alias altogether and let ppl use `tagged_locator('workflow')` after tagging all workflows with ... `workflow`?

Reply:
Good idea, let's do it. I very much prefer this over a named autowiring alias in this case. Let's document this properly and ppl will discover it.

Reply:
Done 👍🏼